src/web/lib/components/App/index.scss

```diff
@@ -24,7 +24,11 @@
     background-position: center center;
     background-repeat: no-repeat;
     background-size: 164px 164px;
-    padding-bottom: 22px;
     flex: 1;
+
+    
+    @media only screen and (max-width: 2500px){
```

Comment:
What's up with this arbitrary 2500px maximum width?

If anything, I would expect the condition to be based on the height.

Reply:
okay

Reply:
pls have a review